docs/plugins/outputs/circonus.asciidoc

```diff
@@ -59,16 +59,10 @@ All values will be passed through `event.sprintf`
 
 Example:
 [source,ruby]
------
   ["title":"Logstash event", "description":"Logstash event for %{host}"]
------
-
 or
-
 [source,ruby]
------
   ["title":"Logstash event", "description":"Logstash event for %{host}", "parent_id", "1"]
------
 
```

Comment:
REGRESSION:  The additional formatting around code samples is required for asciidoctor.